### ISSUE TITLE
Toolbar: Add bold, italic, and underline buttons

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#1636]
Add bold, italic, and underline buttons to the Toolbar. 

Click one of the buttons to apply the appropriate style to the currently selected text.

If no text is selected, apply the style to the whole cursor value.

**Comment by kradchenko** [comment]
Hi @raineorshine! Is this issue still relevant? I would like to work on it.

**Comment by raineorshine** [comment]
Yes, it is!

Let me know if you need any help getting set up!

**Comment by kradchenko** [comment]
Okay, great! Thanks. I will work on this issue and then create PR.

**Comment by kradchenko** [comment]
Hi @raineorshine, I have an issue running the project with commands `npm start` and `npm run start:expo`. Below are screenshots of my errors.

<img width="641" alt="Screenshot 2022-10-25 at 14 16 22" src="https://user-images.githubusercontent.com/47353157/197771624-00451387-94bc-4da0-b52c-b3519d10335e.png">

<img width="643" alt="Screenshot 2022-10-25 at 14 20 27" src="https://user-images.githubusercontent.com/47353157/197771635-38c1627d-ab48-400d-ab00-0e348c6d42ad.png">

**Comment by raineorshine** [comment]
Since your compilation error is not related to this issue, please create a new issue. I can offer some ideas there.

**Comment by kradchenko** [comment]
Hi @raineorshine. My bad on this compilation error. I used `npm` instead of `yarn` so it installed packages and created a new `package-lock.json`. With `yarn` it works fine.

**Comment by kradchenko** [comment]
I've created PR (#1665) addressing this issue. I've basically just created new buttons/shortcuts, then added a keyboard shortcut for each button and created a reducer function for each button, where I've used the already implemented `toggleAttribute` function. Right now these buttons work like toggle buttons, which can bold, underline or change `Thought` to italic style. I am not sure if it is the solution you wanted to achieve. It would be cool if the user could e.g. bold only selected text, but I think it would be quite hard to achieve.

cc @raineorshine
